packages/titanium-single-select/src/titanium-single-select.ts

```diff
@@ -364,7 +372,8 @@ export class TitaniumSingleSelectElement extends LitElement {
       border-top: 0;
       border-bottom-left-radius: 4px;
       border-bottom-right-radius: 4px;
-      box-shadow: 0 4px 6px 0 rgba(32, 33, 36, 0.28);
+      margin-top: -1px;
```

Comment:
This little gem is to make the input-container and search-suggestions overflap.  Without it you get a thin  visible 1px gap between the two.  You can see this in production now.  It was bugging me so I added this.